README.md

```diff
@@ -2,7 +2,7 @@
 
 # Monitoring Eclipse Che with Prometheus
 
-[![Contribute](https://che.openshift.io/factory/resources/factory-contribute.svg)](https://che.openshift.io/f?url=https://github.com/redhat-developer/che-monitoring)
+[![Contribute](https://www.eclipse.org/che/factory-contribute.svg)](https://che.openshift.io/f?url=https://github.com/redhat-developer/che-monitoring)
```

Comment:
what about renaming the image to `contribute.svg` IMO factory is the Che 6 term

Reply:
OK :)